File: articles/the-unknown-the-variable-and-islamic-mathematical-philosophy/fortran/square_completion.f90
program square_completion_example
  implicit none
  real :: b, c, completion
  b = 10.0
  c = 39.0
  completion = (b / 2.0) ** 2
  print *, "completion_term=", completion
  print *, "completed_rhs=", c + completion
end program square_completion_example

File: articles/the-unknown-the-variable-and-islamic-mathematical-philosophy/haskell/SquareCompletion.hs
module Main where

main :: IO ()
main = do
  let b = 10.0
      c = 39.0
      completion = (b / 2.0) ** 2
  putStrLn ("completion_term=" ++ show completion)
  putStrLn ("completed_rhs=" ++ show (c + completion))
